sdk/signalr/signalr/samples/broadcasting.ts

```diff
@@ -0,0 +1,24 @@
+import { HubClient } from "../";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const chatHub = new HubClient(process.env.SIGNALR_CONNECTION_STRING!, "chat");
+
+async function main() {
+  // send a text message to the entire hub
+  await chatHub.broadcast("Hi there!");
```

Comment:
I know this is just a draft but is the intention that a HubClient _is_ a GroupClient as well? Or are there some options that only apply to GroupClients?

Are there nuances?